apps/meteor/server/services/user/service.ts

```diff
@@ -0,0 +1,19 @@
+import { ServiceClassInternal } from '@rocket.chat/core-services';
+import type { IUserService } from '@rocket.chat/core-services';
+import { Users } from '@rocket.chat/models';
+
+import { getMaxLoginTokens } from '../../lib/getMaxLoginTokens';
+
+export class UserService extends ServiceClassInternal implements IUserService {
+	protected name = 'user';
+
+	async ensureLoginTokensLimit(uid: string): Promise<void> {
```

Comment:
should I create an unit test for this? I'd have to mock `Users.findAllResumeTokensByUserId` which is a big part of the whole function..

I guess I should though 🤔